designer/settings/base.py

```diff
@@ -169,12 +169,12 @@
 # Detailed information at: https://docs.djangoproject.com/en/dev/ref/settings/
 SESSION_COOKIE_NAME = 'designer_sessionid'
 CSRF_COOKIE_NAME = 'designer_csrftoken'
-LANGUAGE_COOKIE_NAME = 'designer_language'
+LANGUAGE_COOKIE_NAME = 'openedx-language-preference'
 # END COOKIE CONFIGURATION
 
 # AUTHENTICATION CONFIGURATION
 LOGIN_URL = '/login/'
-LOGOUT_URL = '/logout/'
+LOGOUT_URL = 'http://localhost:18808/logout/'
```

Comment:
I'd love to know more about why we specify a port here.  Should we do it for login as well? Or is the relative path the better pattern?

Reply:
I do think this should probably just be the relative URL